portal-web/docroot/html/taglib/ui/page_iterator/lexicon/start.jsp

```diff
@@ -253,8 +253,8 @@ NumberFormat numberFormat = NumberFormat.getNumberInstance(locale);
 					for (int i = 2; i < ((initialPages > (cur - 1)) ? cur - 1 : initialPages); i++) {
 					%>
 
-						<li>
-							<a class="dropdown-item" href="<%= _getHREF(formName, namespace + curParam, i, jsCall, url, urlAnchor) %>" onclick="<%= forcePost ? _getOnClick(namespace, curParam, i) : "" %>"><span class="sr-only"><liferay-ui:message key="page" /></span><%= i %></a>
+						<li class="<%= ((cur - 3) > 1) ? "" : "page-item" %>">
```

Comment:
Hey @jorgediaz-lr I didn't quite understand this conditional, but would this be for when the element has a DropDown?

Reply:
Yes, the style must be different if we are or we aren't in the dropdown case.

Before [LPS-131514](https://issues.liferay.com/browse/LPS-131514) we have the correct style when we are not in the dropdown but the style of dropdown was incorrect.
After that LPS change, see  https://github.com/brianchandotcom/liferay-portal/pull/101523 the dropdown style is correct, but when there is no dropdown, the vertical alignment is wrong.

To avoid this, it is necessary a different style depending on the case, using the same condition than https://github.com/liferay-frontend/liferay-portal/blob/8a32b941e96d0fe440f333f90c7c416e55ed8d74/portal-web/docroot/html/taglib/ui/page_iterator/lexicon/start.jsp#L240 line so we will have the correct style for both regular and dropdown cases.

Reply:
I see, thanks for clarification @jorgediaz-lr!! On the Clay side components, we can always leave the `page-item` in the `li` tag and when we have DropDown we add the `dropdown-item`, that works fine too.